src/core/tsi/ssl_transport_security.c

```diff
@@ -44,6 +44,9 @@
 
 #include <openssl/bio.h>
 #include <openssl/err.h>
+#if defined(OPENSSL_IS_BORINGSSL)
```

Comment:
It looks like `#include <openssl/crypto.h>` would work in both cases. Can you confirm?